build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -258,17 +263,34 @@ String _entryPointJs(String bootstrapModuleName) => '''
   var mainUri = _currentDirectory + "$bootstrapModuleName";
 
   if (typeof document != 'undefined') {
+    // Load stack trace mapper
```

Comment:
Both this script and the `require.js` script would ideally only get injected once (assuming that does work with `require.js`). Maybe assign an ID to these and query to see if they already exist before adding them?

Reply:
Good idea, I can do that 👍